src/data/markdown/docs/20 jslib/01 jslib/01 aws/07 SQSClient.md

```diff
@@ -0,0 +1,59 @@
+---
+title: 'SQSClient'
+head_title: 'SQSClient'
+description: 'SQSClient enables interaction with the AWS Simple Queue Service (SQS)'
+excerpt: 'SQSClient allows interacting with the AWS Simple Queue Service (SQS)'
+---
+
+`SQSClient` interacts with the AWS Simple Queue Service (SQS). With it, the user can send messages to specified queues and list available queues in the current region. `SQSClient` operations are blocking. k6 recommends reserving their use for the [`setup`](/using-k6/test-lifecycle/) and [`teardown`](/using-k6/test-lifecycle/) stages as much as possible.
+
+Both the dedicated `sqs.js` jslib bundle and the all-encompassing `aws.js` bundle include the `SQSClient`.
+
+### Methods
+
+| Function                                                         | Description                                          |
+| :--------------------------------------------------------------- | :--------------------------------------------------- |
+| [`sendMessage`](/javascript-api/jslib/aws/sqsclient/sendmessage) | Delivers a message to the specified queue.           |
```

Comment:
```suggestion
| [`sendMessage`](/javascript-api/jslib/aws/sqsclient/sqsclient-sendmessage) | Delivers a message to the specified queue.           |
```

Reply:
Good catch ✋🏻 
I need to find a way to get the linkinator tool to work locally...

Reply:
No you're right. I just checked and it doesn't work on localhost. I also have no idea why, it might be something related to how Gatsby serves locally, because I also just tried it on localserver for my hugo-built personal site, and it did work :thinking: